### PR TITLE
Remove or update references to NOSE customization

### LINKED ISSUE #6: Remove NOSE ini files and instructions
the next version of NOSE will have the updated ini files included, so they don't need to be part of smozROM anymore

### PR DESCRIPTION
**Pull request by AdamCatalyst**
- Remove or update references to NOSE customization
- updated NOSE update.html to HTML5
- readme and PY version & year updates

### CODE REVIEW

File: smozROM.py [hunk 1]
```diff
@@ -38,37 +38,7 @@ def debug(str):
...
 def printHeader():
-    print("\n  SmozROM Hack Applier v1.0.1 - smozoma 2018-10\n")
-
-def clearScreen(): 
-    if os.name == 'nt':
-        os.system('CLS')
-    else:
-        os.system('clear')
-
-def wait_key():
```
Comment (smozoma) [inline smozROM.py]: These functions need to be restored or it won't work (clearScreen and wait_key)

**Review by smozoma — CHANGES_REQUESTED** [review]

File: NOSE update.html [hunk 2]
```diff
@@ -1,97 +1,73 @@
...
-program directory, go into the '<span style="font-weight: bold;">NOSE
-Update</span>' directory.<br>
-  </li>
-  <li>Copy the two files, '<span style="font-weight: bold;">hextext.ini</span>'
-and
-'<span style="font-weight: bold;">roms.ini</span>'</li>
-  <li>Paste these files into the '<span style="font-weight: bold;">ini</span>'
-directory
-of
+<body style="margin: 0 auto; padding: 2em 2.5em; max-width:808px;">
```
Comment (smozoma) [inline NOSE update.html]: Can yo do me a favour and put "border: 1px solid;" back in? it looks weird without a border.

**Review by smozoma — CHANGES_REQUESTED** [review]

File: smozROM.py (hunk 1 above)
Comment (AdamCatalyst) [reply smozROM.py]: addressed in new pull request.

File: NOSE update.html (hunk 2 above)
Comment (AdamCatalyst) [reply NOSE update.html]: yer border is back. adjusted padding to match.

**Review by smozoma — APPROVED** [review]
thanks!

**Pull request merged**